garak/generators/openai.py

```diff
@@ -185,6 +185,9 @@ def _call_model(
             "stop": self.stop,
         }
 
+        if _config.run.seed is not None:
+            create_args["seed"] = _config.run.seed
+
```

Comment:
```suggestion
            "stop": self.stop,
            "seed": self.seed,
        }
```

The loop that filters `create_args` below would remove `seed` if it set to `None`.

This may require that `self.seed` be set in `__init__` to avoid an `AttributeError` though.

Reply:
thanks. constructor now sets `self.seed` from the `config_root` it consumes.